### PR TITLE
#177 removing clientlibs not needed and causing errors in the code.

### LINKED ISSUE #177: Address js error while loading home page
Following error is seen while loading the home page
```
clientlibs.min.js:58 Uncaught TypeError: Cannot read properties of null (reading 'value')
    at HTMLDocument.<anonymous> (clientlibs.min.js:58:83)
    at k (jquery.min.js:99:93)
    at Object.fireWith [as resolveWith] (jquery.min.js:100:343)
    at ready (jquery.min.js:104:123)
```

### PR DESCRIPTION
**Pull request by vdua**
- the clientlib require certain element in the page which are not present in the Franklin site.

Fix #177

Test URLs:
- Before: https://main--nedbank--hlxsites.hlx.live/
- After: https://issue_177--nedbank--hlxsites.hlx.live/

### CODE REVIEW

File: blocks/header/nav-utils.js [hunk 1]
```diff
@@ -29,20 +34,26 @@ function appendScripts(doc) {
     let script = document.createElement('script');
     if (item.src) {
       const url = new URL(item.src);
-      if (REPLACE_SCRIPTS.has(url.pathname)) {
+      if (IGNORE_SCRIPTS.has(url.pathname)) {
```
Comment (sdmcraft) [inline blocks/header/nav-utils.js:37]: Nit-pick : Maybe we can do this check upfront just after line#34. That way we can avoid all the `null` checks for `script`.

**Review by sdmcraft — COMMENTED** [review]
Looks like hamburger drill down has stopped working after this change.

Uploading Screen Recording 2023-04-11 at 4.39.35 PM.mov…

File: blocks/header/nb-clientlibs/scripts/navigation/clientlibs.js [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+function resetMenu() {
+  $('.menu-item').removeClass('selectedMenu');
+  $('.menu-item:nth(0)').addClass('selectedMenu');
+  $('.nbd-first_level_side_menu').addClass('displayHide');
+  $('.nbd-first_level_side_menu:nth(0)').removeClass('displayHide');
+}
+let layerArr = [];
+$(document).ready(() => {
+  $(() => {
+    // const g = $('.nbd-hamburger-menu-mob .nbd-hamburger-top-bar').offset().top;
```
Comment (vdua) [inline blocks/header/nb-clientlibs/scripts/navigation/clientlibs.js:86]: the code that needs to be removed is commented out for now. We may need this when the green bar at the top for mobile menu/hamburger is implemented.

File: blocks/header/nb-clientlibs/scripts/navigation/clientlibs.js [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+      for (var m = k[l]; m.charAt(0) === ' ';) { m = m.substring(1); }
+      if (m.indexOf(g) === 0) { return m.substring(g.length, m.length); }
+    }
+    return '';
+  }
+  const p = document.querySelector('.cookiealert');
+  let k = document.querySelector('.acceptcookies');
+  const r = document.querySelector('#takeMeToOldSite');
+  const u = document.querySelector('.nbd-alert-pop');
+  // const w = document.getElementById('domainName').value;
```
Comment (vdua) [inline blocks/header/nb-clientlibs/scripts/navigation/clientlibs.js:236]: did not find the use of this anywhere in this file.

File: blocks/header/nb-clientlibs/scripts/navigation/clientlibs.js [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+  localStorage.getItem('secondaryNavItemSelected');
+  // const v = $('.nbd-navbar-desktop-wrapper .secondarynav').offset().top;
```
Comment (vdua) [inline blocks/header/nb-clientlibs/scripts/navigation/clientlibs.js:238]: the secondary nav is implemented differently in franklin.

File: blocks/header/nb-clientlibs/scripts/navigation/clientlibs.js [hunk 5]
```diff
@@ -0,0 +1,294 @@
...
+  // $('.modal-backdrop.show').css('top', '0');
```
Comment (vdua) [inline blocks/header/nb-clientlibs/scripts/navigation/clientlibs.js:239]: this is the backdrop that appears when search is clicked. Again this is different in franklin.

File: blocks/header/nb-clientlibs/scripts/navigation/clientlibs.js (hunk 2 above)
Comment (vdua) [reply blocks/header/nb-clientlibs/scripts/navigation/clientlibs.js:86]: This green bar doesn't appear in franklin.
image

Comment (sdmcraft) [reply blocks/header/nb-clientlibs/scripts/navigation/clientlibs.js:86]: Yes this is being addressed in https://github.com/hlxsites/nedbank/pull/197

**Review by sdmcraft — APPROVED** [review]
Thanks @vdua . Based on my limited testing, I could not find anything in particular which may have stopped or changed working after your proposed changes. I think we are good to merge. One this you may want to check before doing this is to check that your changes work well with #197 as well in which @jindaliiita is taking care of the hamburger back button in the mobile view.

**Comment by vdua** [comment]
@jindaliiita I have verified that the back button is working in the mobile and desktop view, with your changes merged with this PR. Anything else that needs to be tested ?

**Comment by jindaliiita** [comment]
> @jindaliiita I have verified that the back button is working in the mobile and desktop view, with your changes merged with this PR. Anything else that needs to be tested ?

@vdua Are my changes merged in your branch ? because i am still not seeing my changes.

Screenshot 2023-04-18 at 9 59 10 AM

**Comment by vdua** [comment]
@jindaliiita No, i have verified it locally. Do you want me to merge the main branch, someone would need to approve the changes again

**Comment by jindaliiita** [comment]
> @jindaliiita No, i have verified it locally. Do you want me to merge the main branch, someone would need to approve the changes again

Will do once you merge the main branch here

**Comment by jindaliiita** [comment]
@vdua For Hamburger, i have checked mobile view and desktop view and its seems fine to me.
Thanks

**Pull request merged**